<launch>
	<!-- Arguments -->
	<arg name="yolo_model" 			default="tiny-yolo-voc"/>
	<arg name="yolo_weights_path" 	default="$(find darknet_rsl)/weights"/>
	<arg name="yolo_cfg_model" 		default="$(arg yolo_model).cfg"/>
	<arg name="yolo_weights_model" 	default="$(arg yolo_model).weights"/>
	
	<!-- Start darknet and ros wrapper -->
	<node pkg="darknet_rsl" type="darknet_rsl" name="darknet_rsl" output="screen">
	  <param name="cfg_model" 			value="$(arg yolo_cfg_model)" />
	  <param name="weights_model" 		value="$(arg yolo_weights_model)" />
	  <param name="weights_path" 		value="$(arg yolo_weights_path)" />
	  <param name="camera_topic_name" 	value="/camera/image_raw" />
	  <param name="object_threshold" 	value="0.2" />
	</node>
	
	<!--<node name="republish" type="republish" pkg="image_transport" output="screen" 	args="compressed in:=/front_camera/image_raw raw out:=/camera/image_raw" /> -->
</launch>
